<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <arg name="move_group" default="front_bots"/>
  <arg name="ee_link1" default="a_bot_robotiq_85_tip_link"/>
  <arg name="ee_link2" default="b_bot_robotiq_85_tip_link"/>
  <arg name="ros_rate" default="0.1"/>
  
  <!--  This loads the robot description and starts up moveit -->
  <!-- <include file="$(find o2as_moveit_config)/launch/o2as_moveit_planning_execution.launch">
    <arg name="sim" value="true"/>
  </include> -->
    
  <!-- This calls the ROS node we have written, using the parameters above -->
  <node name="ExampleLoopMoveit" pkg="o2as_moveit_planner" type="2_bot_example_moveit" respawn="false" output="screen">
    <param name="move_group" value="$(arg move_group)"/>
    <param name="ee_link1" value="$(arg ee_link1)"/>
    <param name="ee_link2" value="$(arg ee_link2)"/>
    <param name="ros_rate" value="$(arg ros_rate)"/>
  </node>
  
</launch>
